File: easy/sum-of-all-odd-length-subarrays/sum-of-all-odd-length-subarrays.py
class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        length = len(arr)
        sum = 0
        for i in range(length):
            coeff = ((i + 1) * (length - i) + 1) // 2
            sum += arr[i] * coeff
        return sum

File: easy/sum-of-all-odd-length-subarrays/sum-of-all-odd-length-subarrays.js
/**
 * @param {number[]} arr
 * @return {number}
 */
var sumOddLengthSubarrays = function (arr) {
  let sum = 0;
  for (let i = 0; i < arr.length; i++) {
    const coeff = Math.floor(((i + 1) * (arr.length - i) + 1) / 2);
    sum += arr[i] * coeff;
  }
  return sum;
};
